openadapt/crud.py

```diff
@@ -384,3 +393,11 @@ def get_window_events(recording: Recording) -> list[WindowEvent]:
         list[WindowEvent]: A list of window events for the recording.
     """
     return _get(WindowEvent, recording.timestamp)
+
+
+def newSession() -> None:
```

Comment:
Please use camel_case.

Also what do you think about naming this more descriptively, e.g. `create_new_db_session`?